public/app/features/alerting/unified/Receivers.tsx

```diff
@@ -29,12 +49,17 @@ const Receivers: FC = () => {
   const isRoot = location.pathname.endsWith('/alerting/notifications');
 
   const configRequests = useUnifiedAlertingSelector((state) => state.amConfigs);
+  const contactPointsStateRequest = useUnifiedAlertingSelector((state) => state.contactPointsState);
 
   const {
     result: config,
     loading,
     error,
   } = (alertManagerSourceName && configRequests[alertManagerSourceName]) || initialAsyncRequestState;
+
+  const { result: contactPointsState } =
```

Comment:
This is a bit unclear. Do we expect to get the same response from two different requests?

Reply:
initialAsyncRequestState is the default value